python-main.js

```diff
@@ -307,7 +307,7 @@ function translations() {
     'use strict';
     // These values must be valid language codes
     // https://www.w3.org/TR/REC-html40/struct/dirlang.html#langcodes
-    var validLangs = ['en', 'es', 'pl', 'hr'];
+    var validLangs = ['en', 'es', 'pl', 'hr', 'hk', 'zh-CN'];
```

Comment:
Is `hk` the right language code as specified in the ISOs listed in https://www.w3.org/TR/REC-html40/struct/dirlang.html#langcodes ?

Reply:
Hong Kong dialect isn't listed in the ISO list of language codes. `hk` is what we use for microbit.org `zh-HK` is the code listed for crowdin. In this instance, I'm following the patter for microbit.org

https://en.wikipedia.org/wiki/IETF_language_tag#ISO_3166-1_and_UN_M.49 states we should use a script subtag rather than a regional one and gives the example of `zh-hant` for Hong Kong Chinese.

Either way, I think we need to be consistent across Foundation products.

Reply:
We are adding this value to `<html lang="xx">`, so if hk is not a valid code, then that might affect accessibility. Sounds like this should be `'zh-HK'`? Isn't that what microbit.org uses?

Reply:
As we use a custom language, we've set this as `hk`![Screenshot 2019-12-11 at 08 36 36](https://user-images.githubusercontent.com/31242877/70604939-71e32000-1bf1-11ea-9922-3459369fbe59.png), 
![Screenshot 2019-12-11 at 08 38 55](https://user-images.githubusercontent.com/31242877/70605091-b8387f00-1bf1-11ea-89a1-4a566b4b17a3.png)

so I think we either be consistent, or we raise an issue for to update crowdin for microbit.org too

Reply:
We should be consistent, but not if means being broken.
We should use the right language code and update the microbit.org website as well.

Reply:
Done and opened https://github.com/microbit-foundation/microbit-i18n/issues/1970